source/postprocess/volume_of_fluid_statistics.cc

```diff
@@ -73,40 +75,34 @@ namespace aspect
 
       std::vector<double> global_volume_of_fluid_vol_sums(n_volume_of_fluid_fields);
 
-      Utilities::MPI::sum (local_volume_of_fluid_vol_sums, this->get_mpi_communicator(), global_volume_of_fluid_vol_sums);
+      Utilities::MPI::sum (local_volume_of_fluid_vol_sums, this->get_mpi_communicator(),
+                           global_volume_of_fluid_vol_sums);
 
       std::ostringstream output;
       output.precision(3);
 
       for (unsigned int f=0; f<n_volume_of_fluid_fields; ++f)
         {
-          std::string col_name = "Global volume of fluid volumes for " +
-                                 this->get_volume_of_fluid_handler().name_for_field_index(f);
+          const std::string col_name = "Global volume of fluid volumes for " +
+                                       this->get_volume_of_fluid_handler().name_for_field_index(f);
           statistics.add_value (col_name, global_volume_of_fluid_vol_sums[f]);
 
           output << global_volume_of_fluid_vol_sums[f];
 
           if (f+1 < n_volume_of_fluid_fields)
             output << "/";
 
-          // also make sure that the other columns filled by the this object
-          // all show up with sufficient accuracy and in scientific notation
-          {
-            const std::string columns[] = { col_name
-                                          };
-            for (unsigned int i=0; i<sizeof(columns)/sizeof(columns[0]); ++i)
-              {
-                statistics.set_precision (columns[i], 8);
-                statistics.set_scientific (columns[i], true);
-              }
-          }
+          // also make sure that the columns filled by the this object
+          // show up with sufficient accuracy and in scientific notation
+          statistics.set_precision (col_name, 8);
+          statistics.set_scientific (col_name, true);
         }
-
-      return std::pair<std::string, std::string> ("Global volume of fluid volumes (m^3):",
-                                                  output.str());
     }
+
+    return {"Global volume of fluid volumes (m^3):", output.str()};
```

Comment:
In other places in the code, we use std::make_pair() to return a pair.

Reply:
True. We needed to, because this syntax is C++11 and so we haven't been able to use it before. 

Do you want me to go back to the old syntax, or allow me this language upgrade?

Reply:
No, the upgrade is fine! 
The line itself should go before the close bracket though ;-)